packages/turf-hex-grid/test.ts

```diff
@@ -55,7 +55,7 @@ test("hex-grid", (t) => {
 });
 
 test("grid tiles count", (t) => {
-  const bbox1 = require(directories.in + "bbox1.json").bbox;
```

Comment:
😮

Reply:
Thought this would be ok. Is there a better way to read in a test fixture?

Reply:
Yeah I like the change, I was surprised that we were using require for this.

Reply:
It was a different time :)